pallets/external-validator-slashes/src/lib.rs

```diff
@@ -485,6 +491,84 @@ impl<T: Config> OnEraStart for Pallet<T> {
     }
 }
 
+impl<T: Config> tp_traits::OnEraEnd for Pallet<T> {
+    fn on_era_end(era_index: EraIndex) {
+        // Send up to SLASH_PAGE_SIZE slashes per era
```

Comment:
So the slashes get delayed for at least 1 era? I don't like this, why can't we send them in the next block?